src/api/db/migrate/20170905081525_add_has_secure_password_support.rb

```diff
@@ -0,0 +1,9 @@
+class AddHasSecurePasswordSupport < ActiveRecord::Migration[5.1]
+  def change
+    rename_column :users, :password, :password_old
```

Comment:
Hm, what about ``deprecated_password``?

Reply:
Hm.. I'd need to change a lot of code only for renaming the column name. I think `password_old` is also fine. Is this considered as a nitpick?

Reply:
I don't really like this name and it should only be a simple find and replace, shouldn't it?

@openSUSE/open-build-service what about a second opinion?

Reply:
Still against it. If I `old` I think is ok to use here. Otherwise we would have names like:
* `password_deprecated`
* `password_deprecated_hash_type`
* `password_deprecated_salt`

This I think is way to long...

Reply:
Beside the "hard" work of re-replacing all the "old" texts for the fields...

I also don't like "old" because it sounds like a backup to me... Maybe we can use "md5" as it was the old algorithm... 🤔

Reply:
Then please anything else than old. Old sounds more like a backup but not that it is actually deprecated. Old has a different meaning for me at least.

Reply:
Also the other way around sounds better:
```
deprecated_password
deprecated_password_hash_type
deprecated_password_salt
deprecated_password_equals?
```

What I don't like about ``md5`` suggestion is that it also doesn't show the "state", the password is actually deprecated! But I'm also open for other ideas!

Reply:
@mdeniz I disagree with `deprecated`. It makes all methods and instance variables very long. I don't like that.

I'm not against changing it at all but please make a proposal then.

`md5` is not possible since it also could be something else not only `md5`.

Reply:
My vote is for deprecated. Although old makes sense in conversation (also because we know exactly what is going on here) I think if another new programmer came along and saw these "password_old" columns it would be much less clear what the intention was.

Deprecated is much more precise and explains exactly whats going on. See the definition:

> be deprecated (chiefly of a software feature) be usable but regarded as obsolete and best avoided, typically because it has been superseded.

https://en.oxforddictionaries.com/definition/deprecate

Also I dont think its a problem at all that "deprecated" is longer than "old".

Reply:
OK, I'll gonna change it.